Remotion/Development/UnitTests.Compilation/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,21 @@
+// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
```

Comment:
This file seems to be a leftover from the old project name.

Reply:
@a-ctor am confused. What did you find here in the license header?

Reply:
Hmm. Maybe it was displayed incorrectly I thought I saw double line comments `// // License ...` in one place.

Reply:
me too, just not here. at least in the code.